app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.test.js

```diff
@@ -202,7 +202,7 @@ describe('addEthereumChainHandler', () => {
 
   describe('if a networkConfiguration for the given chainId already exists', () => {
     describe('if the proposed networkConfiguration has a different rpcUrl from the one already in state', () => {
-      it('updates the network with a new networkConfiguration and switches to it', async () => {
+      it('updates the network with a new networkConfiguration', async () => {
```

Comment:
why are we changing this behavior? It seems odd to be able to add a rpc endpoint but have not be on it immediately after